model/storage-private/src/main/java/org/keycloak/storage/datastore/DefaultExportImportManager.java

```diff
@@ -1242,7 +1242,6 @@ public static ComponentModel convertFedProviderToComponent(String realmId, UserF
         model.setFullSyncPeriod(fedModel.getFullSyncPeriod());
         model.setPriority(fedModel.getPriority());
         model.setChangedSyncPeriod(fedModel.getChangedSyncPeriod());
-        model.setLastSync(fedModel.getLastSync());
```

Comment:
I don't know when this is actually called ... is it called when converting some very old realm imports, or is it still in use?